share/smack/lib/smack.c

```diff
@@ -1047,13 +1047,16 @@ void __SMACK_decls(void) {
   DECLARE(BUILTIN_RMODE_CONVERSION, i8, bvdouble, $si2fp,(_ to_fp 11 53));
   DECLARE(BUILTIN_RMODE_CONVERSION, i1, bvdouble, $si2fp,(_ to_fp 11 53));
 
-  #if BUILD_64
+  #if defined(__LP64__) || defined(_LP64) || defined (_WIN64)
```

Comment:
Totally minor thing, but please still fix it. You have an extra space before `(_WIN64)`.